src/appointments.ts

```diff
@@ -113,16 +113,31 @@ async function proceedWithACode(page: Page, impfCode: string) {
     return false;
   }
 
-  if (
-    (!appointmentWarning && !appointmentText) ||
-    (appointmentText &&
-      !appointmentText.includes("stehen leider keine Termine zur") &&
-      !appointmentText.includes("Termine werden gesucht"))
-  ) {
-    // appointments available!!!
-    debug("Appointments available!!");
-    debug("DEBUG: appointmentText=%s", appointmentText);
-    return true;
+  // if (
+    // (!appointmentWarning && !appointmentText) ||
+    // (appointmentText &&
+      // !appointmentText.includes("stehen leider keine Termine zur") &&
+      // !appointmentText.includes("Termine werden gesucht"))
+  // ) 
+  
+  
```

Comment:
Could you please remove the commented code?

Reply:
Sure, done!

Reply:
> Were you able to test the approach? I see from a screenshot that Uhr is shown in the available time fields so this should work fine if the site does not change that.

@marcoklein, thanks for reviewing. Yes, I tested it. It works and I could not see any issues, unless the website changes as you mentioned.